repository/repository.go

```diff
@@ -114,11 +116,17 @@ type UpdateApplication struct {
 	Name                 string                `json:"name,omitempty"`
 	Status               AppStatus             `json:"status,omitempty"`
 	PayPlanType          PayPlanType           `json:"payPlanType,omitempty"`
+	FirstDateSurpassed   time.Time             `json:"firstDateSurpassed,omitempty"`
```

Comment:
Is this field required here? I couldn't find it being used anywhere, especially since there is an update function specific to the `firstDateSurpassed` field.

Reply:
It is used in the `UpdatApplication` func, that field can also be updated from the normal update func. The other special func is to able to update the fields in various records

Reply:
OK, then we need to refactor to fix [this general issue](https://github.com/pokt-foundation/portal-api-go/issues/26) later; not a blocker for this PR though, thanks.